### DISCUSSION TITLE
auto adjust SQL-editor window's name when loading or saving

### DISCUSSION THREAD

**Discussion by Rolf61** [pawelsalawa/sqlitestudio#4560] [Ideas]
I wonder if it would make sense to implement an automatic change of the name of an "unnamed"SQL-editor window e.g. "SQL editor 2" to the file name of the file used, when save-sql-to-file or load-sql-from-file is executed?

**Comment by pawelsalawa** [comment]
Good idea. Created #4561 for this.
